common/src/main/scala/org/genivi/sota/marshalling/AkkaHttpMarshallingSupport.scala

```diff
@@ -0,0 +1,17 @@
+/**
+  * Copyright: Copyright (C) 2016, ATS Advanced Telematic Systems GmbH
+  * License: MPL-2.0
+  */
+package org.genivi.sota.marshalling
+
+import java.time.OffsetDateTime
+import akka.http.scaladsl.unmarshalling.{FromStringUnmarshaller, Unmarshaller}
+import scala.concurrent.Future
+
+trait AkkaHttpMarshallingSupport {
```

Comment:
Think this could be replaced with `Unmarshaller.strict(OffsetDateTime.parse)`.

And should be named `unmarshalling` not marshalling?